doc/releases/changelog-dev.md

```diff
@@ -400,4 +402,4 @@ Mudit Pandey,
 Matthew Silverman,
 Jay Soni,
 David Wierichs,
-Justin Woodring,
+Justin Woodring, Minh Chau
```

Comment:
Feel free to move yourself to just below Astral Cai (it's alphabetic based on last name) :smile:

Reply:
fixed in 3bb4b7cb7bf5e7ef5125fe0c831ec4c552646132

Reply:
feel free to resolve this @trbromley :)